container-search/src/main/java/com/yahoo/search/query/properties/QueryProperties.java

```diff
@@ -93,6 +94,12 @@ private static Map<CompoundName, GetterSetter> createPropertySetterMap() {
         addDualCasedRM(map, Matching.POST_FILTER_THRESHOLD, GetterSetter.of(query -> query.getRanking().getMatching().getPostFilterThreshold(), (query, value) -> query.getRanking().getMatching().setPostFilterThreshold(asDouble(value, 1.0))));
         addDualCasedRM(map, Matching.APPROXIMATE_THRESHOLD, GetterSetter.of(query -> query.getRanking().getMatching().getApproximateThreshold(), (query, value) -> query.getRanking().getMatching().setApproximateThreshold(asDouble(value, 0.05))));
         addDualCasedRM(map, Matching.TARGET_HITS_MAX_ADJUSTMENT_FACTOR, GetterSetter.of(query -> query.getRanking().getMatching().getTargetHitsMaxAdjustmentFactor(), (query, value) -> query.getRanking().getMatching().setTargetHitsMaxAdjustmentFactor(asDouble(value, 20.0))));
+        map.put(CompoundName.fromComponents(Ranking.RANKING, Ranking.MATCHING, Matching.WEAKAND, WeakAnd.STOP_WORD_LIMIT),
+                GetterSetter.of(query -> query.getRanking().getMatching().getWeakAnd().getStopWordLimit(),
+                               (query, value) -> query.getRanking().getMatching().getWeakAnd().setStopWordLimit(asDouble(value, 1.0))));
+        map.put(CompoundName.fromComponents(Ranking.RANKING, Ranking.MATCHING, Matching.WEAKAND, WeakAnd.ADJUST_TARGET),
+                GetterSetter.of(query -> query.getRanking().getMatching().getWeakAnd().getAdjustTarget(),
+                               (query, value) -> query.getRanking().getMatching().getWeakAnd().setAdjustTarget(asDouble(value, 0.0))));
```

Comment:
The default should be 1.0, meaning the feature is turned off.

Reply:
Updated. Not sure what the semantics are of the default value here, since we already explicitly test that _not_ providing the query property returns a `null` `Double` value, i.e. implicitly turned off.